cmd/appleauth/appleauth.go

```diff
@@ -45,13 +47,15 @@ func AuthorizeWithApple(writer http.ResponseWriter, request *http.Request) {
 	if appleDevTokenReqErr != nil {
 		http.Error(writer, appleDevTokenReqErr.Error(), http.StatusInternalServerError)
 		log.Println(appleDevTokenReqErr.Error())
+		errClient.LogErrReq(appleDevTokenReqErr, appleDevTokenReq)
```

Comment:
I'm going to just leave one comment here, but this really applies to all these logging scenarios. I believe it should be something like this:

1. Write error back to the client: `http.Error(writer, appleDevTokenReqErr.Error(), http.StatusInternalServerError)` (Would also love your thoughts on maybe a better way to decide what we should send back in regards to value ie: 200, 404, 500, etc.)
2. Call our log client that will print to console && send to Google Cloud